modin/core/dataframe/pandas/exchange/dataframe_protocol/utils.py

```diff
@@ -0,0 +1,297 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
```

Comment:
do we need these utils if we're not using them at all? maybe we should put them in another PR, when the `from_dataframe` function will be fully implemented and be used somewhere?

Reply:
Yes, let's put it in a separate PR.

Reply:
removed